plugins/plugin-core-graphing/lib/GraphingPlugin.ts

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2020-2023 Delft University of Technology and SynTest contributors
+ *
+ * This file is part of SynTest Framework - SynTest Core Graphing Plugin.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  Encoding,
+  ListenerPlugin,
+  ListenerInterface,
+  CONFIG,
+} from "@syntest/core";
+import { GraphingListener } from "./GraphingListener";
+import Yargs = require("yargs");
+import { mkdirSync } from "fs";
+
+/**
+ * This example plugin logs the program state at the start of the initialization phase of the program.
```

Comment:
Comment from the example plugin. This should be updated

Reply:
Fixed